lakefile.lean

```diff
@@ -48,7 +48,7 @@ require «doc-gen4» from git "https://github.com/leanprover/doc-gen4" @ "main"
 
 require std from git "https://github.com/leanprover/std4" @ "main"
 require Qq from git "https://github.com/leanprover-community/quote4" @ "master"
-require aesop from git "https://github.com/leanprover-community/aesop" @ "master"
+require aesop from git "https://github.com/leanprover-community/aesop" @ "nightly-testing"
```

Comment:
I assume this needs changing before merging?

Reply:
Oops, thank you, missed that one!

Reply:
Fixed now.